### DISCUSSION TITLE
multithreaded

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#46] [Ideas]
Did you look for using the hyperthreading or multicore feature of x86?
Could be awesome to have screen_redraw or keyboard/mouse management or audio playback running on a second proc!
https://github.com/duanev/oz-x86-32-asm-003

**Comment by VendelinSlezak** [comment]
Multithread would be really nice, but so far I did not invest any time to learn how to do it, so now I am not looking for it.
